modules/3d-tiles/test/lib/utils/load-utils.js

```diff
@@ -1,13 +1,14 @@
 // This file is derived from the Cesium code base under Apache 2 license
 // See LICENSE.md and https://github.com/AnalyticalGraphicsInc/cesium/blob/master/LICENSE.md
 
-import {fetchFile, parse} from '@loaders.gl/core';
-import {Tileset3DLoader, Tileset3D} from '@loaders.gl/3d-tiles';
+import {fetchFile, load} from '@loaders.gl/core';
+import {Tiles3DLoader} from '@loaders.gl/3d-tiles';
+import {Tileset3D} from '@loaders.gl/tiles';
 
 export async function loadTileset(t, tilesetUrl) {
   try {
     // Load tileset
-    const tileset = await parse(fetchFile(tilesetUrl), Tileset3DLoader);
+    const tileset = await load(tilesetUrl, Tiles3DLoader);
```

Comment:
Are these changes covered by unit tests?

Reply:
this is load-utils for loading datasets in unit testing.